etc/gramps.profile

```diff
@@ -0,0 +1,54 @@
+# Firejail profile for gramps
+# Description: genealogy program
+# This file is overwritten after every install/update
+# Persistent local customizations
+include gramps.local
+# Persistent global definitions
+include globals.local
+
+# Allow python (blacklisted by disable-interpreters.inc)
+#noblacklist ${PATH}/python2*
+noblacklist ${PATH}/python3*
+#noblacklist /usr/lib/python2*
+noblacklist /usr/lib/python3*
+#noblacklist /usr/local/lib/python2*
+noblacklist /usr/local/lib/python3*
+
+noblacklist ${HOME}/.gramps
+whitelist ${HOME}/.gramps
```

Comment:
Move and add:
```
include disable-xdg.inc

mkdir ${HOME}/.gramps
whitelist ${HOME}/.gramps
include whitelist-common.inc
include whitelist-var-common.inc

apparmor
```

Note: When build a whitelisting-profile you can't store files in dirs like Documents, need gramps this?

Reply:
@curiosity-seeker I didn't say something about the noblacklist line ;)
leave it